driver-core/src/main/com/mongodb/client/model/FindOptions.java

```diff
@@ -608,6 +610,32 @@ public FindOptions snapshot(final boolean snapshot) {
         return this;
     }
 
+    /**
+     * Returns the exhaust.
+     *
+     * Determines whether the returned cursor from operation execution will be an exhaust cursor.
+     *
+     * @return the exhaust
+     * @since 3.9
+     */
```

Comment:
Add `@mongodb.server.release 4.2` to the two new methods here.

Reply:
Done